packages/xstate-viz/package.json

```diff
@@ -33,7 +33,6 @@
     "@types/styled-components": "^4.4.0",
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
-    "tsdx": "^0.11.0",
```

Comment:
had to remove this as it has caused bizarre problems with `vue-jest` and we actually don't really even use `@xstate/viz` here, so better to remove this for now and try to bring it back when we need to

I've put my findings of the issue in Yarn's issue here: https://github.com/yarnpkg/yarn/issues/6285#issuecomment-751402589